.github/workflows/test_user_guide.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
       - name: setup micromamba
         uses: mamba-org/setup-micromamba@v1
         with:
-          environment-file: ci/envs/312-latest.yaml
```

Comment:
I am using some of the geopandas 1.0 api (union_all). GeoPandas will be released sooner than momepy.